src/repositories/request-repository.ts

```diff
@@ -267,7 +269,17 @@ export class RequestRepository extends Repository<Request> {
             )
           }
 
-          // TODO(NET-1623): Add alert here that we marked expired and processing requests as READY
+          const expired = requests.filter((request) => request.createdAt < anchoringDeadline)
```

Comment:
this function is so huge already.  Can you pull this metrics reporting out into a helper function?